server/controllers/images/lib/containers.js

```diff
@@ -1,9 +1,7 @@
 const images_ = require('lib/images')
 const putImage = require('./put_image')
 
-const containerPutImage = (container, fnName) => fileData => {
-  const { id, path } = fileData
-
+const containerPutImage = (container, fnName) => ({ id = 0, path }) => {
```

Comment:
not sure about those optimisation "au poil de cul", sometimes its useful to actually read what the object represent

Reply:
done in 7d341acde